src/test/regress/expected/failure_single_mod.out

```diff
@@ -107,10 +107,11 @@ DETAIL:  server closed the connection unexpectedly
 	This probably means the server terminated abnormally
 	before or while processing the request.
 COMMIT;
+ERROR:  failure on connection marked as essential: localhost:xxxxx
```

Comment:
My understanding is that we'd like to do away with the concept of unhealthy shards, so this change of expected behavior should be supported by updating the comment. Otherwise the connection shouldn't be marked essential